protostar/cheatable_starknet/callable_hint_locals/warp_hint_local.py

```diff
@@ -16,10 +14,10 @@ def _build(self):
 
     def warp(
         self,
-        target_contract_address: RawAddress,
+        target_contract_address: int,
         blk_timestamp: int,
-    ) -> Callable[[], None]:
-        return self.controllers.block_info.cheat(
+    ):
+        self.controllers.block_info.cheat(
```

Comment:
Why is this method called `cheat`? Couldn't we use something like `set_block_timestamp` + `set_block_number` instead?

Reply:
It's a legacy thing, it was copied over. If you feel like it, we can change the structure in this PR.

Reply:
https://github.com/software-mansion/protostar/pull/1429/commits/3f458f11e2f42aa50d0590e2b5ecb963d10185e1